stake-pool/cli/src/output.rs

```diff
@@ -0,0 +1,179 @@
+                     pool.address.to_string(),
+                     pool.manager.to_string(),
+                     pool.total_lamports.to_string(),
+                     pool.pool_token_supply.to_string(),
+                     pool.validator_list.len().to_string()
+            ).ok();
+        }
+        writeln!(f, "Total number of pools: {}", &self.pools.len().to_string())
+    }
+}
+
+impl QuietDisplay for CliStakePools {}
+impl VerboseDisplay for CliStakePools {}
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CliStakePool {
+    pub address: String,
+    pub manager: String,
+    pub staker: String,
+    pub stake_deposit_authority: String,
+    pub stake_withdraw_bump_seed: u8,
+    pub max_validators: u32,
+    pub validator_list: Vec<CliStakePoolValidator>,
+    pub reserve_stake: String,
+    pub pool_mint: String,
+    pub manager_fee_account: String,
+    pub token_program_id: String,
+    pub total_lamports: u64,
+    pub pool_token_supply: u64,
+    pub last_update_epoch: u64,
+    pub lockup: CliStakePoolLockup,
+    pub epoch_fee: CliStakePoolFee,
+    pub next_epoch_fee: Option<CliStakePoolFee>,
+    pub preferred_deposit_validator_vote_address: Option<String>,
+    pub preferred_withdraw_validator_vote_address: Option<String>,
+    pub stake_deposit_fee: CliStakePoolFee,
+    pub stake_withdrawal_fee: CliStakePoolFee,
+    pub next_stake_withdrawal_fee: Option<CliStakePoolFee>,
+    pub stake_referral_fee: u8,
+    pub sol_deposit_authority: Option<String>,
+    pub sol_deposit_fee: CliStakePoolFee,
+    pub sol_referral_fee: u8,
+    pub sol_withdraw_authority: Option<String>,
+    pub sol_withdrawal_fee: CliStakePoolFee,
+    pub next_sol_withdrawal_fee: Option<CliStakePoolFee>,
+    pub last_epoch_pool_token_supply: u64,
+    pub last_epoch_total_lamports: u64,
+
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CliStakePoolValidator {
+    pub active_stake_lamports: u64,
+    pub transient_stake_lamports: u64,
+    pub last_update_epoch: u64,
+    pub transient_seed_suffix_start: u64,
+    pub transient_seed_suffix_end: u64,
+    pub status: String,
+    pub vote_account_address: String,
+}
+
+impl From<ValidatorStakeInfo> for CliStakePoolValidator {
+    fn from(v: ValidatorStakeInfo) -> Self {
+        Self {
+            active_stake_lamports: v.active_stake_lamports,
+            transient_stake_lamports: v.transient_stake_lamports,
+            last_update_epoch: v.last_update_epoch,
+            transient_seed_suffix_start: v.transient_seed_suffix_start,
+            transient_seed_suffix_end: v.transient_seed_suffix_end,
+            status: match v.status {
+                StakeStatus::Active => "Active",
+                StakeStatus::DeactivatingTransient => "DeactivatingTransient",
+                StakeStatus::ReadyForRemoval => "ReadyForRemoval",
+            }.to_string(),
```

Comment:
Did you mean to `impl From<StakeStatus> for CliStakePoolValidatorStakeStatus` and use it here?